prime-router/src/main/kotlin/cli/tests/TestReportStream.kt

```diff
@@ -1050,12 +1123,109 @@ class StracPack : CoolTest() {
         waitABit(5 * options.submits, environment) // SWAG: wait extra seconds extra per file submitted
         reportIds.forEach {
             passed = passed and
-                examineLineageResults(reportId = it, receivers = listOf(redoxReceiver), totalItems = options.items)
+                pollForLineageResults(reportId = it, receivers = listOf(redoxReceiver), totalItems = options.items)
         }
         return passed
     }
 }
 
+class Parallel : CoolTest() {
+    val n = 50
```

Comment:
I got this error running the parallel test:
` Got exception while dispatching to schema tx/tx-covid-19, and rcvr ignore.HL7_NULL
prime_dev_1     | [2021-09-08T15:55:55.324Z] Internal Error: multiple item report cannot be written as a single HL7 message
prime_dev_1     | [2021-09-08T15:55:55.324Z] Result: Internal Error: multiple item report cannot be written as a single HL7 message
prime_dev_1     | [2021-09-08T15:55:55.324Z] Exception: Internal Error: multiple item report cannot be written as a single HL7 message
prime_dev_1     | [2021-09-08T15:55:55.324Z] Stack: java.lang.IllegalStateException: Internal Error: multiple item report cannot be written as a single HL7 message
`